packages/aws-rfdk/test-python.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -eou pipefail
```

Comment:
nit: I had thought this needed to be:

```suggestion
set -euo pipefail
```

since the `pipefail` is an arg to the `-o` command-line option of `set`.

Reply:
Oh yeah, my mistake.